### ISSUE TITLE
When using Full Framebuffer Emulation, pausing and resuming in flycast causes temporary screen corruption

### ISSUE THREAD

**Issue by kihato** [flyinghead/flycast#1079]
Platform / OS / Hardware: 
Windows11 / i7-10700F / GTX 1660 SUPER

Github hash: e94babe

Occurs with graphics APIs except Vulkan for the following games.

D2: loaded photo
Densya de Go 2 3000-Bandai: Screen display during play
Sweet Season: OMAKE - CG Mode - transition effect

It happens just by pausing it, but it's not permanent and seems to return to normal over time.

**Comment by flyinghead** [comment]
Yes, this is a known issue. It's intermittent and depends on the GPU and API used. It's due to a synchronization problem between the renderer and the emulator threads when handling vblank.

Switching to single-thread mode fixes the issue (obviously).
